src/main/java/core/basesyntax/Writer.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Writer {
+    public void writeToFile(int[] data, String fileName) {
+        File file = new File(fileName);

```

Comment:
same thing as in reader, about dividing of logic

Reply:
Yes, there is the same line in different methods, but it received different arguments: input and output fileName. Changed names of arguments by more understandible